File: src/main/java/ru/raiffeisen/demoapplication/controllers/UserAuthenticationController.java
package ru.raiffeisen.demoapplication.controllers;

import java.security.Principal;
import java.util.Base64;
import javax.servlet.http.HttpServletRequest;
import org.springframework.web.bind.annotation.CrossOrigin;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@CrossOrigin
public class UserAuthenticationController {
	
	@RequestMapping("/user")
	public Principal user(Principal principal) {
		return principal;
	}
}

File: src/main/java/ru/raiffeisen/demoapplication/controllers/UserAuthenticationController.kt
package ru.raiffeisen.demoapplication.controllers

import org.springframework.web.bind.annotation.CrossOrigin
import java.security.Principal
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@CrossOrigin
class UserAuthenticationController {
    @RequestMapping("/user")
    fun getUser(user: Principal): Principal = user
}
